build.gradle

```diff
@@ -1568,16 +1581,16 @@ project(':test-common') {
   }
 
   dependencies {
+    loggingImplementation log4j2Libs
```

Comment:
I think only core and connect-runtime needs to add  log4j2Libs to `compileOnly` scope.